OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/files/FileChooserService.java

```diff
@@ -67,9 +67,11 @@ public static File getSelectedFile(String extension, String description, DialogT
         int returnVal = 0;
         switch (type) {
             case SAVE:
+                chooser.setDialogTitle("Save As");
```

Comment:
That's the convention that I'm used to. Clicking 'Save' when a new/unsaved file is open in Word, Excel, etc. will open a dialog with the title 'Save As'. 

If you agree with using the convention but think the code is unclear, I can try to make it more readable.  For example, in the 'getFileWithConfirmation' method, I could change the type to SAVEAS when file == null.